### PR TITLE
[WebNN EP] Support several activation ops

### PR DESCRIPTION
**Pull request by Honry**
Support Elu, HardSigmoid, HardSwish, Softplus, Softsign, Tanh.

### CODE REVIEW

**Comment by Honry** [comment]
@fdwr, @guschmue, PTAL, thanks!

**Review by guschmue — APPROVED** [review]

**Comment by fdwr** [comment]
Cool, because I have a local Chromium implementation for:

- elu
- prelu
- leakyRelu
- clamp
- linear
- sigmoid
- hardSigmoid
- hardSwish
- softplus
- softsign
- softmax
- tanh
- abs
- log
- neg

File: onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc [hunk 1]
```diff
@@ -31,33 +31,39 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
...
-      emscripten::val options = emscripten::val::object();
+
+  if (Contains(model_builder.GetFusedActivations(), node.InputDefs()[0]->Name())) {
+    LOGS_DEFAULT(VERBOSE) << op_type << " Node [" << node.Name() << "] fused";
+    output = input;
+  } else {
+    NodeAttrHelper helper(node);
+    emscripten::val options = emscripten::val::object();
+    if (op_type == "Elu") {
+      options.set("alpha", helper.Get("alpha", (float)1.0));
```
Comment (fdwr) [inline onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc:42]: Why not just `1.0f`? The C-style cast looks weird.

File: onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc [hunk 2]
```diff
@@ -67,15 +73,27 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
 // Operator support related.
 
 int ActivationOpBuilder::GetMinSupportedOpSet(const Node& /* node */) const {
-  // All ops opset 5- uses consumed_inputs attribute which is not supported for now.
+  // HardSwish is available since opset 14, and the reset ops opset 5-
```
Comment (fdwr) [inline onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc:76]: reset ops?

**Review by fdwr — COMMENTED** [review]
Nits. Thanks Wanming.

File: onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc (hunk 1 above)
Comment (Honry) [reply onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc]: Ah, just quickly copy/paste, fixed.

File: onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc (hunk 2 above)
Comment (Honry) [reply onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc]: 'rest ops', :p

File: onnxruntime/core/providers/webnn/builders/impl/argmax_min_op_builder.cc [hunk 3]
```diff
@@ -93,3 +93,4 @@ void CreateArgMaxMinOpBuilder(const std::string& op_type, OpBuilderRegistrations
 
 }  // namespace webnn
 }  // namespace onnxruntime
+f)
```
Comment (fdwr) [inline onnxruntime/core/providers/webnn/builders/impl/argmax_min_op_builder.cc:96]: Typo?

**Review by fdwr — APPROVED** [review]
Thanks Wanming. ⭐

**Pull request merged**